scripts/benchmarks/benches/compute.py

```diff
@@ -43,14 +45,14 @@ def setup(self):
         run(f"cmake --build {build_path} -j", add_sycl=True)
 
         self.built = True
-        self.bins = os.path.join(build_path, 'bin')
 
 class ComputeBenchmark(Benchmark):
     def __init__(self, bench, name, test):
         self.bench = bench
         self.bench_name = name
         self.test = test
         super().__init__(bench.directory)
+        self.setup()
```

Comment:
why?

Reply:
Removed (after reverting the options.rebuild logic).